results-mieb/openai/clip-vit-large-patch14/no_model_name_available/no_revision_available/RenderedSST2.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dataset_revision": "66b9a461eda025201dd147e5f390f5984c33643a",
+  "evaluation_time": 27.11316442489624,
+  "kg_co2_emissions": null,
```

Comment:
We might as well run everything with codecarbon, that makes reporting it in the paper easier.